NGUIButtonInsetsExample/ViewController.swift

```diff
@@ -14,13 +14,13 @@ import UIKit
 final class TimerPool {
     static let shared = TimerPool()
     private var timers: [Timer] = []
-
+    
     func addTimer(_ block: @escaping () -> Void) {
         timers.append(Timer.scheduledTimer(withTimeInterval: 0.1, repeats: true, block: { _ in
             block()
         }))
     }
-
+    
```

Comment:
One last thing - looks like we added unnecessary whitespace here. Could we remove that? I believe there's a setting in Xcode preferences to trim trailing whitespace. Thanks!

Reply:
Sure